cmd/endorse.go

```diff
@@ -38,9 +41,12 @@ import (
 // endorseCommand stores the common flag values for the endorse subcommand that isn't directly
 // represented in endorse.Context.
 type endorseCommand struct {
-	AddSnp   bool
-	AddTdx   bool
-	UefiPath string
+	Storage             storagei.Client
```

Comment:
Please comment that this is the filesystem abstraction for reading files whose paths are passed in as flags.